tools/cli/templates/projects/webpack/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "gameface-webpack-project",
+  "version": "1.0.0",
```

Comment:
In the vite template it's `private` and the version is `0.0.0`. Can you change one or the other so it's more in sync?